bin/SleepRayTune.py

```diff
@@ -57,20 +57,20 @@ def SleepRayTune(config, cfg: DictConfig, num_epochs, num_gpu):
 
 @hydra.main(config_path="../config", config_name="config")
 # model=m2, dataset=m2, training=m2 in terminal
-def main(cfg: DictConfig, num_epochs=100, gpus_per_trial=1):
+def main(cfg: DictConfig, num_epochs=200, gpus_per_trial=1):
```

Comment:
Would you name this file using just lower case letters an underscores? That's the convention

Reply:
Can `num_epochs` be set in the config file?

Reply:
I would agree with this , also can you name it `n_epochs` for consistency. Thanks!